clockwork_web/templates/settings.html

```diff
@@ -103,28 +103,35 @@ <h1>{{ gettext("User settings %(mila_email_username)s", mila_email_username=mila
 			<div class="col-12">
 
 				<dl class="row">
-					<!--
 					<dt class="col-6">
+						{% if current_user.clockwork_api_key %}
+							<form>
+								<div class="form-group">
+									<label>{{ gettext("CLOCKWORK_API_KEY:") }}</label>
```

Comment:
It's a bit strange to have this as though it's a value that requires translating into many languages. There's no harm in having a `gettext` call, but it could lead someone to ask why or to wonder if alternative names exist.

Reply:
If it seems to be a minor question, I would preferred to let the `gettext` in order to keep it as a translation question (while filling the translation pairing). (But it is not indisputable at all)